dpdata.py

```diff
@@ -159,6 +160,12 @@ def __fixup_nomail_flag(self):
                 # Empty out NOMAIL_REASON if NOMAIL not set
                 dp_donorrecord['NOMAIL_REASON'] = ''
 
+    def __copy_spouse_email(self):
```

Comment:
This logic used to be implemented along with similar logic for empty parent 1 name in  create_dp_donorrecord(district_record, school_year)
However the logic seems to have been deleted when the code was refactored. Shouldn't we reinstate this code there? Any reason we should give preferential "scrub" treatment to email but not name?